pkg/config/application_kubernetes.go

```diff
@@ -94,6 +94,10 @@ type KubernetesDeploymentInput struct {
 	// Automatically reverts all deployment changes on failure.
 	// Default is true.
 	AutoRollback *bool `json:"autoRollback,omitempty" default:"true"`
+
+	// Automatically create a new namespace if it does not exist.
+	// Default is false.
+	AutoCreateNamespace *bool `json:"autoCreateNamespace,omitempty" default:"false"`
```

Comment:
```suggestion
	AutoCreateNamespace bool `json:"autoCreateNamespace,omitempty"`
```
We only need to use the type `*bool` in case of the default value is set to true, or else the bool type on golang has the default value is set to false 😄 
ref: https://github.com/creasty/defaults/issues/30#issuecomment-942435138 (we're using this package for defaults values in configuration object)

Reply:
oh cool thanks, I couldn't imagined that from the time being!